packages/wxt/src/core/utils/building/__tests__/find-entrypoints.test.ts

```diff
@@ -668,27 +668,42 @@ describe('findEntrypoints', () => {
     expect(entrypoints[0]).toEqual(expected);
   });
 
-  it('should not allow multiple entrypoints with the same name', async () => {
+  it('should ignore other index files when index.html exists', async () => {
+    globMock.mockResolvedValueOnce([
+      'popup/index.html',
+      'popup/index.ts',
+      'popup/index.css',
+    ]);
+
+    const entrypoints = await findEntrypoints();
+    console.log(entrypoints);
+
+    expect(entrypoints).toHaveLength(1);
+    expect(entrypoints[0]).toMatchObject({
+      name: 'popup',
+      type: 'popup',
+    });
+  });
+
+  it('should not allow a file entrypoint and directory entrypoint to have the same name', async () => {
     globMock.mockResolvedValueOnce([
-      'options/index.html',
-      'options/index.jsx',
       'popup.html',
       'popup/index.html',
       'popup/index.ts',
-      'ui.html',
+      'other.ts',
+      'other/index.ts',
```

Comment:
Slightly tweaked the inputs to this test, so it tests both HTML and script entrypoints.